storefront/app/helpers/spree/filters_helper.rb

```
@@ -26,19 +26,23 @@ def default_storefront_products_for_filters
+        distinct.
+        count
+      end
     end
 
     def product_single_filter_aggregations
-      @product_single_filter_aggregations ||= default_storefront_products_for_filters.
-                                              joins(variants_including_master: :option_values).
-                                              group("#{Spree::OptionValue.table_name}.id").
-                                              distinct.
-                                              count
+      @product_single_filter_aggregations ||= Rails.cache.fetch(["product-single-filter-aggregations", default_storefront_products_for_filters], expires_in: 1.day) do
```

RuboCop (1.75.5)

[convention] Prefer single-quoted strings when you don't need string interpolation or special symbols.

(Style/StringLiterals)